package.json

```diff
@@ -36,7 +36,6 @@
     "typescript": "3.6.4"
   },
   "dependencies": {
-    "deep-equal": "1.1.0",
```

Comment:
need install `fast-deep-equal`

Reply:
Oh thanks a lot! I don’t know why it was missed :)
Will fix